bitnami/redis/templates/configmap.yaml

```diff
@@ -83,11 +83,11 @@ data:
       sentinel client-reconfig-script {{ .Values.sentinel.masterSet }} /opt/bitnami/scripts/start-scripts/push-master-label.sh
     {{- end }}
     {{- if .Values.auth.acl.sentinel }}
-    {{- $userSecret := .Values.auth.acl.userSecret -}}
     # Sentinel ACL configuration, only for users with password
```

Comment:
The diff looks a bit messy here, this is basically:
* move the comment one line up
* replace `}}` with `-}}` in the following three lines